packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/edit/VSphereCredentialsEdit.tsx

```diff
@@ -19,7 +19,7 @@ import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
 
 import { EditComponentProps } from '../BaseCredentialsSection';
 
-export const VSphereCredentialsEdit: React.FC<EditComponentProps> = ({ secret, onChange }) => {
+export const VSphereCredentialsEdit: React.FC<EditComponentProps> = ({ secret, url, onChange }) => {
   const { t } = useForkliftTranslation();
 
   const user = safeBase64Decode(secret?.data?.user || '');
```

Comment:
```suggestion
export const VSphereCredentialsEdit: React.FC<EditComponentProps> = ({ secret, onChange }) => {
  const { t } = useForkliftTranslation();

  const user = safeBase64Decode(secret?.data?.user || '');
  const url = safeBase64Decode(secret?.data?.url || '');
```
it's nicer to use the URL directly from the  secret, ( instead of creating two sources for the URL value ), this approach will also remove the need to pass a new prop